requirements.txt

```diff
@@ -1,29 +1,29 @@
 alembic==1.0.0
 async-timeout==4.0.2
-attrs==22.2.0
+attrs==23.1.0
 Babel==2.9.1
 Beaker==1.11.0
 bleach==3.3.0
-boto3==1.26.73
-botocore==1.29.73
+boto3==1.26.115
+botocore==1.29.115
 certifi==2022.12.7
 cffi==1.15.1
 chardet==3.0.4
 ckan==2.9.8
-ckanext-datajson==0.1.15
+ckanext-datajson==0.1.18
```

Comment:
This is weird; normally we bump this ourselves. Why have we gotten so out of date?

Reply:
We had harvesting updates that didn't effect inventory and that's why we never looked to update it here.